package.json

```diff
@@ -75,5 +75,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "dependencies": {}
```

Comment:
This was probably accidentally caught up in a commit, I don't think it seems like it should impact the functionality of the pr. Please revert this change.